lib/streamlit/runtime/media_file_manager.py

```diff
@@ -192,10 +195,10 @@ def del_expired_files(self) -> None:
         for file_id, imf in list(self._files_by_id.items()):
```

Comment:
nit: there's a bunch of places where `imf` is used as an abbreviation for `InMemoryFile`, which has been renamed to `MediaFile`, so I guess the abbreviations should be swapped to `mf`

Reply:
what did you just call me

Reply:
😱